app/bounty_requests/views.py

```diff
@@ -60,7 +61,16 @@ def bounty_request(request):
             return JsonResponse({'error': 'Invalid JSON.'}, status=400)
 
         model = result.save(commit=False)
+
+        gh_org_api_url = 'https://api.github.com/orgs/' + model.github_org_name
+        gh_org_api_resp = requests.get(url=gh_org_api_url).json()
+
+        gh_org_email = ''
+        if gh_org_api_resp is not None and gh_org_api_resp['email'] is not None:
```

Comment:
If the org didn't setup an email is failing. 
If the org doesn't have an email just keep the old flow and send just to vivek